### ISSUE TITLE
SyntaxError: missing ) after argument list

### ISSUE THREAD

**Issue by swatty007** [suchipi/jest-node-nw-example#3]
Hey
Im having some troubles to get this gem running. 
I tried a couple of things and ended up at one point at the same spot as trusktr in ticket #1 that I could run the test but didnt got any output.

Hope somebody can help me to get some actual output.

I installed the package on a Windows 10 machine with node v8.12.0, yarn 1.17.3 and nwjs v0.40.1sdk set in windows path variable.

If I run **yarn test** I get the following error:
```
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^

SyntaxError: missing ) after argument list
```

If I run the test command from jest directly like so:
`node_modules/.bin/jest --env=jest-environment-node-nw`

it fails on window is undefined instead.
```
FAIL  .\demonstration.test.js
  ● Test suite failed to run

    ReferenceError: window is not defined
```

Any idea whats missing on my end?

**Comment by tonetechnician** [comment]
Yep, I've also been getting this issue. This seems to be an [issue](https://github.com/facebook/jest/issues/4751) with Windows and the jest.cmd. This issue is still open and it seems people have lost interest in trying to solve it. 

I'm running a different version though:

node: 12.12.0
yarn: 1.19.1
nwjs: 0.42.0-sdk

One way to get past that error is to change the packages/tests/package.json file test command to 

```
  "scripts": {
    "test": "node-nw ../node_modules/jest/bin/jest.js --env=jest-environment-node-nw --watch"
  },
```

But this results in further errors:

```
 FAIL  packages\tests\demonstration.test.js
  ● Test suite failed to run

    TypeError: Cannot set property 'console' of undefined

      at module.exports (node_modules\jest-util\build\setGlobal.js:17:37)
```

This is the same error I get when trying to run the jest.cmd directly as @swatty007 has done.

Trying to see if I can fix this as I really would love to get jest going in my nw app.

**UPDATE:**

So turns out that I was just referencing the wrong /bin/jest.js file by not using a double '../'. Noob work!

For clarity sake:

1. Clone the repo
2. `yarn install`
2. Edit packages/tests/package.json script tag to:

```
  "scripts": {
    "test": "node-nw ../../node_modules/jest/bin/jest.js --env=jest-environment-node-nw --watch"
  },
```

4. `yarn test` 

And it all works. 

Hope this helps!
